utility/BUILD.bazel

```diff
@@ -47,7 +48,7 @@ android_library(
     testonly = True,
     custom_package = "org.oppia.android.util",
     manifest = ":test_manifest",
-    resource_files = glob(["src/test/res/**"]),
+    resource_files = glob(["src/test/res/**/*.xml"]),
```

Comment:
Does this distinction actually matter vs. just globbing everything?

Reply:
I was trying out every other possibility just to check if that regex is matching well or not. I will revert it now.